src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -189,7 +194,7 @@ class EventErrors extends React.Component<Props, State> {
             <EventErrorItem key={errorIdx} error={error} />
           ))}
 
-          {project.features.includes('reprocessing-v2') && (
+          {'features' in project && project.features.includes('reprocessing-v2') && (
```

Comment:
```suggestion
          {project?.features?.includes('reprocessing-v2') && (
```

You could also use optional chaining.

Reply:
I need to differentiate between `AvatarProject` (does not have features prop) and `Project` (has it) with the first clause, and this doesn't work for that.